ui/app/components/secret-engine/configure-ssh.hbs

```diff
@@ -3,84 +3,73 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<form {{on "submit" (perform this.save)}} aria-label="save ssh creds" data-test-configure-form>
-  <div class="box is-fullwidth is-shadowless is-marginless">
```

Comment:
removed lots of extra divs here, also the the box shadow above the input is anti-pattern so I removed it
# before/after
<img width="1719" alt="Screenshot 2024-09-23 at 5 02 02 PM" src="https://github.com/user-attachments/assets/f70bac9c-3a65-4844-a54e-b9424714ba97">